app/src/main/java/com/example/dashcarr/presentation/tabs/settings/PowerSavingMode.kt

```diff
@@ -0,0 +1,28 @@
+package com.example.dashcarr.presentation.tabs.settings
+
+
+object PowerSavingMode {
+    enum class PowerState {
+        AUTO, ON
+    }
+
+    private lateinit var appPowerMode: PowerState
+    private var batteryPowerMode = false
+    fun setAppPowerMode(mode: PowerState) {
+        appPowerMode = mode
+    }
+
+    fun setBatteryMode(mode: Boolean){
+        batteryPowerMode = mode
+    }
```

Comment:
Use consistent naming. I.e. appPowerMode and phonePowerMode
Also pay attention to the names of your getter and setters